tests/acceptance/stepDefinitions/provisioningContext.js

```diff
@@ -178,7 +178,7 @@ Given('these users have been created with default attributes but not initialized
 ) {
   return Promise.all(
     dataTable.rows().map(userId => {
```

Comment:
In order to keep the scope of `map` minimal, you could also introduce another `map` which just does `.map(userIds => userIds.toString())` before the map that deletes the user.